aexpect/remote.py

```diff
@@ -523,10 +527,10 @@ def scp_to_remote(host, port, username, password, local_path, remote_path,
     command = "scp"
     if directory:
         command = "%s -r" % command
-    command += (" -v -o UserKnownHostsFile=/dev/null "
-                "-o StrictHostKeyChecking=no "
-                "-o PreferredAuthentications=password %s "
-                "-P %s %s %s@\[%s\]:%s" %
+    command += (r" -v -o UserKnownHostsFile=/dev/null "
+                r"-o StrictHostKeyChecking=no "
+                r"-o PreferredAuthentications=password %s "
+                r"-P %s %s %s@\[%s\]:%s" %
```

Comment:
well the `r` prefix is only required for the last line, but no big deal...

Reply:
I would say it's better to have it for all of them for better safety and symmetry. Please, unresolve this note if you don't agree.

Reply:
Looking at it again (because I forgot I already have) it is not a regexp, it is just incorrect string. It should be `"-P %s %s %s@\\[%s\\]:%s"` as we want to put there `\[` and not to escape `[`. The result is the same, but it'd be better to use pure strings to avoid confusion in the future, anyway it's not a high priority right now...